packages/select/docs/FilterMultiSelect.stories.tsx

```diff
@@ -72,10 +73,10 @@ export const DefaultKaizenSiteDemo: ComponentStory<
           </>
         )}
       </FilterMultiSelect>
-      <Paragraph variant={"body"}>
-        Items:{" "}
+      <Box mt={4}>
+        <Paragraph variant={"body"}>Items: </Paragraph>{" "}
```

Comment:
```suggestion
        <Paragraph variant="body">Items: </Paragraph>{" "}
```

Nitpick: If we're fixing the line anyway, might as well :)

Reply:
Yep all done :)